### DISCUSSION TITLE
[Announcement] Environment setup for programming assignments

### DISCUSSION THREAD

**Discussion by sujin0529** [prosyslab-classroom/cs348-information-security#380] [Announcements]
This semester, students will use OCaml and several tools for programming assignments.

To help you set up the programming environment, we provide a pre-built docker image and KCLOUD VMs.

For detailed information, please see the [instructions](https://docs.google.com/presentation/d/1bgJH1fSrEczQ9KipIGxsV8GbmtRDhp3boglGEj3WvWE/edit?usp=sharing).

**Comment by tanapthetimid** [comment]
I pulled the docker container prosyslab/classroom and it has Ubuntu 20.04.4 and llvm 13.0.1 unlike in the picture in the instructions which has Ubuntu 18.04.2 and llvm 10.0.0. Is this alright?

**Comment by KAIST-JongchanPark** [comment]
@TanapTheTimid 
Yes. We plan to use Ubuntu 20.04 and llvm 13 for this 2023S course. 
The pictures are old one. We will update it asap.

Thank you.

**Comment by m-spitfire** [comment]
Is it enforced to use llvm 13? I had llvm 14 setup, will it be breaking for upcoming homeworks?

Thanks.

**Comment by KAIST-JongchanPark** [comment]
@m-spitfire 
Please use the given environment if you can.

We will grade your HWs with above setting.

FYI, we will provide KCLOUD VM soon which everything is already installed.

**Comment by crossedfinger-j** [comment]
I installed Docker and in my terminal using "docker login" logged in. After that "docker pull prosyslab/classrom" but I have an error about 
"
Error response from daemon: pull access denied for prosyslab/classrom, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
"
Is any solution for this error? Thanks.

**Comment by KAIST-JongchanPark** [comment]
It's probably because of typo. Please try with "prosyslab/classroom"
